meteor/server/api/rundownLayouts.ts

```diff
@@ -40,12 +37,11 @@ export function removeRundownLayout (layoutId: RundownLayoutId) {
 	RundownLayouts.remove(layoutId)
 }
 
-const postJsRoute = Picker.filter((req, res) => req.method === 'POST')
-postJsRoute.middleware(bodyParser.text({
+PickerPOST.middleware(bodyParser.text({
```

Comment:
should this middleware still exist here?

Reply:
good find! I fix